content/en/docs/languages/js/serverless.md

```diff
@@ -271,13 +271,12 @@ Add the following to your `package.json`:
     "@google-cloud/functions-framework": "^3.0.0",
     "@opentelemetry/api": "^1.9.0",
     "@opentelemetry/auto-instrumentations-node": "^0.56.1",
-    "@opentelemetry/exporter-trace-otlp-http": "^0.57.2",
-    "@opentelemetry/instrumentation": "^0.57.2",
-    "@opentelemetry/sdk-node": "^0.57.2",
```

Comment:
Note for reviewers: this dependency was removed on purpose - it is not used in the above code so it can be safely dropped, which will reduce install size 🙂